### PR TITLE
Nars-Ian now keeps its mind after rebirth

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Fixes sentient Ian not keeping being sentient after getting transformed into Nars-Ian

## Why It's Good For The Game

Bugfix

## Changelog

:cl:
fix: Fixes sentient Ian not keeping being sentient after getting transformed into Nars-Ian
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
based as hell, is it tested?

**Comment by SuperSlayer0** [comment]
> based as hell, is it tested?

No, but it is litteraly a one line of code change, there... isn't much things not to work

**Comment by LemonInTheDark** [comment]
I do not trust our codebase, could you test it please

**Comment by SuperSlayer0** [comment]
> I do not trust our codebase, could you test it please

Uhhhh, perhaps not now

File: code/modules/mob/living/simple_animal/friendly/dog.dm [hunk 1]
```diff
@@ -552,6 +552,7 @@ GLOBAL_LIST_INIT(strippable_corgi_items, create_strippable_list(list(
 	playsound(src, 'sound/magic/demon_dies.ogg', 75, TRUE)
 	var/mob/living/simple_animal/pet/dog/corgi/narsie/N = new(loc)
```
Comment (tralezab) [inline code/modules/mob/living/simple_animal/friendly/dog.dm:553]: A reason to test stuff I guess with a second commit; You're using a single letter variable, could you make it a bit more descriptive while you're here?

**Review by tralezab — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/friendly/dog.dm (hunk 1 above)
Comment (SuperSlayer0) [reply code/modules/mob/living/simple_animal/friendly/dog.dm:553]: I am not using a single letter variable, it was already there, i just added the mind transfer line. But i can change the variable name, no problem.

Comment (LemonInTheDark) [reply code/modules/mob/living/simple_animal/friendly/dog.dm:553]: you mind doin that?

**Pull request closed without merging**